StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/PaymentSheetIntentConfiguration.swift

```diff
@@ -15,9 +15,9 @@ import Foundation
     /// - Seealso: https://stripe.com/docs/payments/finalize-payments-on-the-server
     struct IntentConfiguration {
 
-        /// Pass this into `intentCreationCallback` to force PaymentSheet to show success and dismiss.
-        /// - Note: Only for advanced users, not required for most integrations.
-        @_spi(STP) public static let FORCE_SUCCESS = "FORCE_SUCCESS"
+        /// Pass this into `intentCreationCallback` to force PaymentSheet to show success, dismiss the sheet without confirming payment, and return a PaymentSheetResult of `completed`.
```

Comment:
Nit: Might not always be a payment, so should we rephrase this?

```suggestion
        /// Pass this into `intentCreationCallback` to force PaymentSheet to show success, dismiss the sheet without confirming the intent, and return a PaymentSheetResult of `completed`.
```

Reply:
Building off this, wdyt about adding even more details about when to use this and what the caveats are?

```
/// Pass this into `intentCreationCallback` to force PaymentSheet to show success, dismiss the sheet without confirming the intent, and return a PaymentSheetResult of `completed`. 
/// - Note: ⚠️ If provided, the SDK performs no action to complete the payment or setup - it doesn't confirm a PaymentIntent or SetupIntent or handle next actions. 
///   You should only use this if your integration can't create a PaymentIntent or SetupIntent. It is your responsibility to ensure that you only pass this value if the payment or set up is successful. 
```

Reply:
@yuki-stripe yeah I like that! adding...